test/simulator/main/org/apache/cassandra/simulator/paxos/PaxosTopologyChangeVerifier.java

```diff
@@ -58,7 +58,7 @@ public void before(Topology before, int[] participatingKeys)
     @Override
     public void after(Topology topologyAfter)
     {
-        afterInternal(topologyAfter.select(topologyBefore.primaryKeys));
+        afterInternal(topologyBefore == null ? topologyAfter : topologyAfter.select(topologyBefore.primaryKeys));
```

Comment:
I was injecting errors into the read write validation logic to see how errors were handled and hit a code path where `topologyBefore` was null... not 100% sure this is correct, so do let me know... it looked correct when reviewing `select` logic